crates/uv/tests/pip_sync.rs

```diff
@@ -127,8 +127,7 @@ fn missing_venv() -> Result<()> {
         ----- stdout -----
 
         ----- stderr -----
-        error: failed to canonicalize path `[VENV]/bin/python3`
-          Caused by: No such file or directory (os error 2)
+        error: No Python interpreters found in virtual environments
```

Comment:
Now this isn't amazing, I guess with `-v` you'd see why your `VIRTUAL_ENV` was ignored. We could add some sort of hint in the future?

Reply:
Since it literally doesn't exist this seems okay for now though